src/packaging/Makefile

```
@@ -16,7 +16,7 @@
 package:
 	mvn -B clean package  -f ../../pom.xml -DskipTests
 
-prepare:
+prepare: package
```

Perhaps this target should depend on the JAR path? That way if the JAR exists, we just do the `prepare` steps. If the JAR file is missing we then go and build it.

```suggestion
prepare: $(REAPER_JAR)
```

We should also change the `all` target to depend on the JAR path in a similar way as well.